templates/aws-tap-entrypoint-new-vpc.template.yaml

```diff
@@ -625,8 +590,8 @@ Resources:
           - !Ref AWS::NoValue
         PublicSubnet1Id: !GetAtt VPCStack.Outputs.PublicSubnet1ID
         RemoteAccessCidr: !Ref RemoteAccessCidr
-        LinuxBastionOS: !Ref LinuxBastionOS
-        LinuxBastionSshPort: !Ref LinuxBastionSshPort
+        LinuxBastionOS: Ubuntu-Server-22.04-LTS-HVM
+        LinuxBastionSshPort: 22
```

Comment:
*suggestion/non-blocking*:
If we opt for removing the ssh port and OS/AMI configuration, we should remove it from those parameters from the "existing vpc" template and remove anything we do in that stack:

Candidates for removal: 
- https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/blob/main/templates/aws-tap-entrypoint-existing-vpc.template.yaml#L64-L68
- https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/blob/main/templates/aws-tap-entrypoint-existing-vpc.template.yaml#L64-L68
- https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/blob/main/templates/aws-tap-entrypoint-existing-vpc.template.yaml#L448-L453
- https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/blob/main/templates/aws-tap-entrypoint-existing-vpc.template.yaml#L441-L447
- https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/blob/main/templates/aws-tap-entrypoint-existing-vpc.template.yaml#L1212-L1213
- https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/blob/main/templates/aws-tap-entrypoint-existing-vpc.template.yaml#L2128
- https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/blob/main/templates/aws-tap-entrypoint-existing-vpc.template.yaml#L2128

There might be more, tho, I didn't do an exhaustive check.